test/Microsoft.TestPlatform.AcceptanceTests/TelemetryTests.cs

```diff
@@ -1,157 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Removing it as of now, beacause in EnvironmentVariable logic has been removed. Will raise in next PR